File: app/src/main/java/com/example/signup/Info2Request.java
package com.example.signup;

import com.android.volley.Response;
import com.android.volley.toolbox.StringRequest;

import java.util.HashMap;
import java.util.Map;

public class Info2Request extends StringRequest {

    final static private String URL = "http://favor531.ivyro.net/Info2Request.php";
    private Map<String, String> map;

    public Info2Request(String Ucareer, String info_ration, String Ulicense, String lovation_work, String uworkTime, String userID, Response.Listener<String> listener)
    {
        super(Method.POST, URL, listener, null);

        map = new HashMap<>();
        map.put("Ucareer",Ucareer);
        map.put("info_ration", info_ration);
        map.put("Ulicense", Ulicense);
        map.put("lovation_work", lovation_work);
        map.put("uworkTime", uworkTime);
        map.put("userID", userID);





    }


    @Override
    protected Map<String, String> getParams()
    {

        return map;
    }
}

File: php소스/Download/Info2Request.php
<?php 
    $con = mysqli_connect("localhost", "favor531", "qwerty123!", "favor531");
    mysqli_query($con,'SET NAMES utf8');

//    public Info2Request(String Ucareer, String info_ration, String Ulicense, String lovation_work, String uworkTime, String userID Response.Listener<String> listener)

    $Ucareer = $_POST["Ucareer"];
    $info_ration = $_POST["info_ration"];
    $Ulicense = $_POST["Ulicense"];
    $lovation_work = $_POST["lovation_work"];
    $uworkTime = $_POST["uworkTime"];
    $userID = $_POST["userID"];

    
    
    



    $statement = mysqli_prepare($con, "INSERT INTO CAREGIVER(Ucareer, info_ration, Ulicense, lovation_work, uworkTime, userID) VALUES (?,?,?,?,?,?)");
    mysqli_stmt_bind_param($statement, "ssssss", $Ucareer, $info_ration, $Ulicense, $lovation_work, $uworkTime, $userID);
    mysqli_stmt_execute($statement);


    $response = array();
    $response["success"] = true;
 
   
    echo json_encode($response);



?>